tests/test_ivflib.py

```diff
@@ -179,7 +179,7 @@ def test_small_data(self):
         ref_lims, ref_D, ref_I = index.range_search(xq, thresh)
         gt_index = faiss.IndexFlat(d)
         gt_index.add(xb)
-        lims, D, I = index.range_search(xq, thresh)
+        lims, D, I = gt_index.range_search(xq, thresh)
```

Comment:
makes sense. NB that usually the ref_XXX is used for the ground truth (means reference)